src/views/recordTable.component.test.tsx

```diff
@@ -110,106 +109,108 @@ describe('Record Table', () => {
   it('calls the correct data fetching hooks on load', () => {
     createView();
 
-    expect(useRecordsPaginated).toHaveBeenCalledWith({
-      page: 0,
-      sort: {},
-      dateRange: {},
-    });
+    expect(useRecordsPaginated).toHaveBeenCalled();
     expect(useRecordCount).toHaveBeenCalled();
     expect(useChannels).toHaveBeenCalled();
   });
 
-  it('updates page query parameter on page change', async () => {
-    data = Array.from(Array(12), (_, i) => generateRecord(i + 1));
-    (useRecordsPaginated as jest.Mock).mockReturnValue({
-      data: data,
-      isLoading: false,
-    });
-    (useRecordCount as jest.Mock).mockReturnValue({
-      data: data.length,
-      isLoading: false,
-    });
-    props.resultsPerPage = 10;
+  it('can sort columns and removes column sort when column is closed', async () => {
```

Comment:
This is an okay test but I looked around and noticed that we don't seem to have any tests that make sure `state.columns.sort` is accurately manipulated whenever we attempt to sort a column. I'd quite like to see this in `columnsSlice.test.tsx` personally but it can go here if you prefer, as long as it's somewhere. 

For example, the check to make sure `state.columns.selectedColumnIds` is correctly manipulated happens in `columnCheckboxes.test.tsx`, instead of `columnsSlice.test.tsx`. This is okay, as long as that test happens somewhere.